webapp/src/sui.tsx

```diff
@@ -1272,13 +1277,15 @@ export class Modal extends data.Component<ModalProps, ModalState> {
     renderCore() {
         const { isOpen, size, longer, basic, className,
             onClose, closeIcon, children, onKeyDown,
-            header, headerIcon, headerClass, headerActions, helpUrl, description,
+            header, headerFn, headerIcon, headerClass, headerActions, helpUrl, description,
             closeOnDimmerClick, closeOnDocumentClick, closeOnEscape,
             shouldCloseOnEsc, shouldCloseOnOverlayClick, shouldFocusAfterRender, overlayClassName, ...rest } = this.props;
         const { marginTop, scrolling, mountClasses } = this.state;
         const isFullscreen = size == 'fullscreen';
         const showBack = isFullscreen && !!closeIcon;
 
+        const resolvedHeader = header ? header : headerFn ? headerFn() : undefined;
```

Comment:
this is the reverse of the button logic. for the buttonfn in coretsx, the function overrides the property not the other way around

Reply:
Nice. will make this one behave the same.